ahcore/transforms/pre_transforms.py

```diff
@@ -218,15 +289,34 @@ class ImageToTensor:
     def __call__(self, sample: DlupDatasetSample) -> dict[str, DlupDatasetSample]:
         tile: pyvips.Image = sample["image"]
         # Flatten the image to remove the alpha channel, using white as the background color
-        tile_ = tile.flatten(background=[255, 255, 255])
+        using_features = False
+
+        if tile.bands > 4:
+            # assuming that more than four bands/channels means that we are handling features
+            using_features = True
+            tile_ = tile
```

Comment:
I don't think we should use pyvips when we have features. In that case torch.Tensor of numpy arrays should be what we are using.

Reply:
A dlup dataset will always return a pyvips image and it does work as currently implemented, but indeed we could think about if we want to change that.